tasks/wait-for-node-ready.yml

```
@@ -1,18 +1,18 @@
 ---
 # for master nodes wait for the master API port to be ready
 - block:
-  - name: OCP | Wait for Node Ready | Determine master API port
+  - name: OCP | Wait for Master Ready | Determine master API port
```

```suggestion
  - name: OCP | Wait for Node Ready | Determine master API port
```
I think the task naming convention that is used here is derived from the task filename, so we should leave it saying Node even though it only applies to masters.

---

Reverted change.